### ISSUE TITLE
Add support for name alias's / ID's to allow for easier referencing

### ISSUE THREAD

**Issue by jamesread** [jamesread/orgviz#1]
For example;

```
Mr Longname
    title: Person with a long name

Alice 
    title: A testing user
    supports -> Mr Longname
```

Consider; 

```
Mr Longname
    title: Person with a long name
    id: MrL

Alice 
    title: A testing user
    supports -> MrL
```

Ideal proposal from Michael Kuehl.

**Comment by mickume** [comment]
exactly what I was thinking ...

make it optional, i.e. check id first, if it doesn't exists assume the real name is meant

**Comment by Fale** [comment]
Hi! Is this an issue due to missing time (so contribution is welcome) or is needed some additional thinking about it, before the implementation can be started?

If it's the first case, I can pick up the ticket!

**Comment by jamesread** [comment]
Hey Fabio! Contributions would be super welcome :-) It's purely that I have not had the time! Thanks for volunteering, here is what I was thinking as a suggested approach; 

- The Person class has some "hard coded" attributes from when I first started writing `orgviz` (eg: `team`, `fullName`, etc), but I switched to a more flexible "dictionary" of attributes, for future expansion, for features like this. So I don't think we need to change anything in the Person class - we can start updating org files with `id:` already. 
- In the `Model` class, create a new method called `findPerson`, which first iterates over `self.people`, using `getAttribute('id')` to see if it can find someone there. Then, if not, use the existing `getPersonByName` method as a "fallback". ie: It defaults to alias silently, and then "falls back" to name. If the person is not found, then the existing exception in `getPersonByName` is thrown as normal.
- Update references from `getPersonByName` to `findPerson` in the other parts of the code :)

Make sense? Thanks for the offer as well! I'll milestone this for the next release.
